### ISSUE TITLE
Update to quiz check

### ISSUE THREAD

**Issue by carriewright11** [jhudsl/OTTR_Template#656]
Noticed that we could add this to our quiz formatting checks... 

- Checking for space between quiz question instructions and question:

https://github.com/fhdsl/AI_for_software_Quizzes/pull/4

- Checking for question mark at the beginning of the question:

https://github.com/fhdsl/Ethical_Data_Handling_for_Cancer_Research_Quizzes/pull/10

- Checking for space after the question before the options:

https://github.com/fhdsl/Ethical_Data_Handling_for_Cancer_Research_Quizzes/pull/6/commits/ce4e4d1d42fbac311ab6f4370646ebad1cb8a015

I think this was a problem too, but not sure -  using ** for bold  or * for italic text: 
https://github.com/fhdsl/Ethical_Data_Handling_for_Cancer_Research_Quizzes/pull/11

**Comment by carriewright11** [comment]
@cansavvy don't know if you want this issue on the main ottr_template repo

**Comment by cansavvy** [comment]
Yeah, I've generally kept all the issues on the same repo so as it make it simpler for tracking. Because sometimes its not clear the solution and what repo might be involved in creating that solution.

**Comment by ehumph** [comment]
@cansavvy @carriewright11 Just found another issue - two quizzes did not have unique IDs but the quiz check was still successful. Threw this error on Leanpub: Step 16/34 Failed: Duplicate quiz ids found. There is more than one quiz with an id of "quiz_ethics". https://github.com/fhdsl/AI_for_software_Quizzes/pull/12

**Comment by carriewright11** [comment]
Also need a space after that question mark before the text of the question starts
https://github.com/fhdsl/AI_for_software_Quizzes/pull/14

**Comment by carriewright11** [comment]
gah curly apostrophes are quotes appear to be an issue!

**Comment by carriewright11** [comment]
I think it would be best to try to make these errors and check them - I thought the quiz repo would have quiz checks as default for a bit earlier and didn't catch that until later - I have an issue about that too and hope to get to that

**Comment by carriewright11** [comment]
apparently if you skip a letter like a c d e then it causes an issue: 
https://github.com/fhdsl/Overleaf_and_LaTeX_for_Scientific_Articles_Quizzes/pull/4/files
